wvd-templates/Create and provision WVD host pool/nestedtemplates/managedDisks-customimagevm.json

```diff
@@ -51,6 +51,13 @@
             },
             "defaultValue": "Standard_A2"
         },
+        "enableAcceleratedNetworking": {
```

Comment:
[Applies to this one, and the other nested templates..] Once we have the value as a parameter here, we should trust it. Let's only check in the mainTemplate.json file.

Reply:
We can apply to all of them, this is supported on Windows 2012 R2 and forward, from gallery or custom images, it leaves the impression that is not supported if you're deploying from the portal directly from the custom image because the function that identifies it cannot open the image on the fly and determine if it is an OS that supports or not, which might be true for Linux VMs since there are outdated versions of Linux that really does not support Mellanox drivers, but it does not make sense for Windows since all supported versions of Windows supports it.

Reply:
I think that the exceptions would be worth it then not leaving the option there at all, most customers will benefit from the option be there, they need to understand what means changing the default value from False to True.